Move /usage/distributed-tracing to /tracing (#9572)
getsentry/sentry-docs: platform-includes/tracing/how-to-use/java.mdx

If you use the current version of our Java SDK, tracing works automatically with the following frameworks:

- [Spring](/platforms/java/guides/spring/tracing/#how-to-use-tracing)
- [Spring Boot](/platforms/java/guides/spring-boot/tracing/#how-to-use-tracing)

> If you use SDK version `6.24.x` or below, you need to <PlatformLink to="/performance/">Set Up Performance</PlatformLink> monitoring for tracing to work. If you do not want to use performance monitoring, you can instead set up <PlatformLink to="/tracing/custom-instrumentation/">Custom Instrumentation</PlatformLink> for distributed tracing.

Reviewer: ```suggestion
If you use SDK version `6.24.x` or below, you need to <PlatformLink to="/performance/">Set Up Performance</PlatformLink> monitoring for tracing to work. If you don't want to use performance monitoring, you can set up <PlatformLink to="/tracing/custom-instrumentation/">Custom Instrumentation</PlatformLink> for tracing instead.
```
